reason-react-native/src/types/Modal_Orientation.rei

```diff
@@ -13,4 +13,12 @@ let landscapeRight: t;
 let portrait: t;
 
 [@bs.inline "portrait-upside-down"]
-let portraitUpsideDown: t;
\ No newline at end of file
+let portraitUpsideDown: t;
+
+type eventValue;
+
+[@bs.inline "landscape"]
+let landscape_: eventValue;
```

Comment:
I am afraid this won't be intuitive. Shouldn't we have a Change module in there or Modal_OrientationChange?

Reply:
I vote for Modal_OrientationChange.

Reply:
Implemented `Modal_OrientationChange`.